index.bs

```diff
@@ -422,1025 +422,901 @@ A variety of additional use cases and configurations are also possible, includin
-external hardware, or a combination of both.
+As was the case in earlier sections, this flow focuses on a use case involving a [=first-factor roaming authenticator=]
+with its own display. One example of such an authenticator would be a smart phone. Other authenticator types are also supported
+by this API, subject to implementation by the [=client platform=]. For instance, this flow also works without modification for the case of
+an authenticator that is embedded in the [=client device=]. The flow also works for the case of an authenticator without
+its own display (similar to a smart card) subject to specific implementation considerations. Specifically, the [=client platform=]
+needs to display any prompts that would otherwise be shown by the authenticator, and the authenticator needs to allow the [=client
+platform=] to enumerate all the authenticator's credentials so that the client can have information to show appropriate prompts.
 
-### Backwards Compatibility with FIDO U2F ### {#sctn-conforming-authenticators-u2f}
 
-[=Authenticators=] that only support the [[#sctn-fido-u2f-attestation]] have no mechanism to store a
-[=user handle=], so the returned {{AuthenticatorAssertionResponse/userHandle}} will always be null.
+### Registration ### {#sctn-sample-registration}
 
-## [WRPS] ## {#sctn-conforming-relying-parties}
+This is the first-time flow, in which a new credential is created and registered with the server.
+In this flow, the [=[WRP]=] does not have a preference for [=platform authenticator=] or [=roaming authenticators=].
 
-A [=[WRP]=] MUST behave as described in [[#sctn-rp-operations]] to obtain all the security benefits offered by this specification. See
-[[#sctn-rp-benefits]] for further discussion of this.
+1. The user visits example.com, which serves up a script. At this point, the user may already be logged in using a legacy
+    username and password, or additional authenticator, or other means acceptable to the [=[RP]=].
+    Or the user may be in the process of creating a new account.
 
+1. The [=[RP]=] script runs the code snippet below.
 
-## All Conformance Classes ## {#sctn-conforming-all-classes}
+1. The [=client platform=] searches for and locates the authenticator.
 
-All [=CBOR=] encoding performed by the members of the above conformance classes MUST be done using the
-[=CTAP2 canonical CBOR encoding form=].
-All decoders of the above conformance classes SHOULD reject CBOR that is not validly encoded
-in the [=CTAP2 canonical CBOR encoding form=] and SHOULD reject messages with duplicate map keys.
+1. The [=client=] connects to the authenticator, performing any pairing actions if necessary.
 
+1. The authenticator shows appropriate UI for the user to select the authenticator on which the new credential will be
+    created, and obtains a biometric or other authorization gesture from the user.
```

Comment:
This looks inaccurate - maybe it used to say the "the _client_ shows appropriate UI..."? Should perhaps be more like:

>The authenticator shows appropriate UI for the user to provide a biometric or other authorization gesture.

Reply:
OK.